rfc/4/comments/1.md

```diff
@@ -0,0 +1,27 @@
+# RFC-4 comment
+
+## Comment author
+
+David Stansby
+
+## Conflicts of interest (optional)
+
+None
+
+## Summary
+
+n/a
+
+## Significant comments and questions
+
+RFC-4 defines anatomical orientation metadata.
+As currently written, it does not restrict this metadata to be present only on spatial axes.
+Because it does not make sense to define an anatomical orientation for non-spatial axes (e.g., channel or time axes), RFC4 should be updated to specific that anatomical orientation data can only be present on spatial axes.
```

Comment:
```suggestion
Because it does not make sense to define an anatomical orientation for non-spatial axes (e.g., channel or time axes), RFC4 should be updated to specify that anatomical orientation data can only be present on spatial axes.
```